runApp.R

```diff
@@ -1,17 +1,3 @@
 
-# Install all the required package for the EMTR function and income explorer app
-pkg <- c("shiny", "rhandsontable", "RColorBrewer", "shinythemes",
-         "plotly", "openxlsx", "data.table", "ggplot2", 
-         "scales", "magrittr", "zoo")
-
-new.pkg <- pkg[!(pkg %in% installed.packages())]
-
-if (length(new.pkg) > 0) {
-  install.packages(new.pkg)
-}
-
-if(packageVersion("shiny") < "1.5.0"){
-  install.packages("shiny")
-}
 
 shiny::runApp()
```

Comment:
I plan to convert this app back to an R package so that the dependencies will be taken care of in the description file under the import field. Also, this install.packages() function does not quite work when uploading it onto the shinyapp.io for hosting.